pkg/identity/identity.go

```diff
@@ -30,6 +32,17 @@ type Identity struct {
 	LabelsSHA256 string `json:"labelsSHA256"`
 }
 
+// IPIdentityPair is a pairing of an IP and the security identity to which that
+// IP corresponds.
+//
+// WARNING - STABLE API
+// This structure is written as JSON to the key-value store. Do NOT modify this
+// structure in ways which are not JSON forward compatible.
+type IPIdentityPair struct {
+	IP net.IP
+	ID NumericIdentity
```

Comment:
For the sake of debugging, it will be incredibly useful to add node and endpoint info. We already have a string representation for this in the API:

`cilium-global:cluster1:nodeX:45234`

Reply:
Where is information like 'nodeX' stored? Is this the same as `nodeName` in `pkg/k8s/init.go:Init()`?

Reply:
On the list of things to add, I have cluster name, node name, endpoint ID. Is there any other identifying information you'd like me to add?

Reply:
I would encode all three of them in a single string:

`cilium-global:default:node1:3333`

```
node.GetLocalNode()
```

You can hardcode the cluster name to the AddressSpace for now. The #2374 PR will add a cluster name do `Node struct`